vespa/application.py

```diff
@@ -105,18 +106,20 @@ def __init__(
         deployment_message: Optional[List[str]] = None,
         cert: Optional[str] = None,
         key: Optional[str] = None,
+        vespa_cloud_secret_token: Optional[str] = None,
         output_file: IO = sys.stdout,
         application_package: Optional[ApplicationPackage] = None,
     ) -> None:
         """
         Establish a connection with an existing Vespa application.
 
-        :param url: Vespa instance URL.
-        :param port: Vespa instance port.
+        :param url: Vespa data plane endpoint.
```

Comment:
`data plane` is a bit confusing for users that want to connect to Vespa instances not on our cloud, for instance a locally running Docker instance?

Reply:
This is just nit picking, but perhaps we should leave it as `endpoint` and `port` - and mention data planes etc in the param description below?

Reply:
Agree

Reply:
Fixed with https://github.com/vespa-engine/pyvespa/pull/573/commits/02ccd75b18f68950496e32ce897daf6b26e55854